src/signals/incident/components/IncidentForm/index.test.js

```diff
@@ -227,9 +231,7 @@ describe('<IncidentForm />', () => {
         next,
       });
 
-      defer(() => {
-        expect(next).toHaveBeenCalled();
-      });
+      expect(next).toHaveBeenCalled();
```

Comment:
Same here; assert that `next` has not been called before any changes to have been applied.